SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -519,7 +516,7 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 
 		ClientConnectionManager = MakeUnique<SpatialGDK::ClientConnectionManager>(SystemEntitySubview, this);
 
-		Dispatcher->Init(Receiver, StaticComponentView, SpatialMetrics, SpatialWorkerFlags);
+		Dispatcher->Init(Receiver, SpatialMetrics, SpatialWorkerFlags);
```

Comment:
cool, how far off killing this are we?

Reply:
Another branch removes receiver from there so I'd say it's just worker flags left